doc/rst/source/datasets/earth-daynight.rst

```diff
@@ -66,6 +66,16 @@ to the geotiff files on the remote server. **Note**: This data set is experiment
 format and delivery is likely to change in the future (e.g., via image tiles).  To make the
 files as small as possible we have also downgraded them from 24-bit to 8-bit indexed images.
 
+Data Updates
```

Comment:
It's better to write it in one file (e.g., `dataset_updates.rst_`) and include it in all other files.

Reply:
you are wiser than me. Will do.